apps/arena_load_test/lib/arena_load_test/game_socket_handler.ex

```diff
@@ -83,6 +86,19 @@ defmodule ArenaLoadTest.GameSocketHandler do
     {:reply, frame, state}
   end
 
+  def terminate(_, state) do
+    case :ets.lookup(:clients, state.client_id) do
+      [{client_id, _}] ->
+        :ets.delete(:clients, client_id)
+
+      [] ->
+        raise KeyError, message: "Client with ID #{state.client_id} doesn't exist."
+    end
```

Comment:
Why not calling the `:ets.delete(:clients, client_id)` directly? do we want to stop everything when we do not find the client in the :clients table due to this custom rise?